### PR TITLE
chore: [CO-393] disable zimbraAdmin webapp connector

### PR DESCRIPTION
**Pull request by keshavbhatt**
- Disable the zimbraAdmin webapp

### CODE REVIEW

File: conf/jetty/jetty.xml.production [hunk 1]
```diff
@@ -754,7 +754,7 @@
     %%comment SERVICE:zimbra,<!--%% ZIMBRAWEBAPPEND -->
 
     <!-- ZIMBRAADMINWEBAPPBEGIN %%comment SERVICE:zimbraAdmin,-->%%
-    <New id="zimbraAdmin" class="org.eclipse.jetty.webapp.WebAppContext">
+    <!--<New id="zimbraAdmin" class="org.eclipse.jetty.webapp.WebAppContext">
```
Comment (Polpetta) [inline conf/jetty/jetty.xml.production:757]: If this part is commented it makes sense to remove it. Is git afterall, we can check what it was in the past commits

**Review by Polpetta — CHANGES_REQUESTED** [review]
Remove commented code

File: conf/jetty/jetty.xml.production (hunk 1 above)
Comment (keshavbhatt) [reply conf/jetty/jetty.xml.production:757]: @Polpetta makes sense, the idea is to un-comment it in production server if the sysadmin wants to re-enable the webapp :)
This is one of the core requirement of the ticket.

**Review by aheeva-yuliya — APPROVED** [review]

File: conf/jetty/jetty.xml.production (hunk 1 above)
Comment (Polpetta) [reply conf/jetty/jetty.xml.production:757]: okok!

**Review by Polpetta — APPROVED** [review]

**Pull request merged**